source/Engine.cpp

```diff
@@ -1623,6 +1623,31 @@ void Engine::MoveShip(const shared_ptr<Ship> &ship)
 		// self-destruct.
 		if(ship->IsDestroyed())
 		{
+			if(!ship->CoreShip().empty())
+			{
+				const Ship *shipToPlace = GameData::Ships().Get(ship->CoreShip());
+				if(shipToPlace->IsValid())
+				{
+					const shared_ptr<Ship> escapeShip = make_shared<Ship>(*shipToPlace);
+					escapeShip->Recharge();
+					escapeShip->SetName(ship->Name());
+					escapeShip->SetGovernment(ship->GetGovernment());
+					if(ship->CorePersonality().IsValid())
+						escapeShip->SetPersonality(ship->CorePersonality());
+					else
+						escapeShip->SetPersonality(ship->GetPersonality());
+					escapeShip->SetHail(*ship->GetHailPhrase());
+					escapeShip->SetSystem(ship->GetSystem());
+					if(ship->GetParent())
+						escapeShip->SetParent(ship->GetParent());
```

Comment:
```suggestion
					if(ship->GetParent() && !escapeShip->GetPersonality()->IsCoward())
						escapeShip->SetParent(ship->GetParent());
```
I think this would be fitting